app/api/dds/v1/auth_providers_api.rb

```diff
@@ -84,6 +84,11 @@ def unsupported_affiliate_search_error!
         affiliate_params = declared(params, {include_missing: false}, [:full_name_contains, :username, :email])
         auth_service = AuthenticationService.find(params[:id])
         unsupported_affiliate_search_error! unless auth_service.identity_provider
+        if ENV['FF_LDAP_PAGINATION']
+          p = paginate(Kaminari.paginate_array(Array.new(500)))
```

Comment:
That's the max number of results passed back from ldap, which is set here:

https://github.com/Duke-Translational-Bioinformatics/duke-data-service/pull/1253/files#diff-a89a3c8956854e179524fb7c9c35a0e9R58

It could be an IdentityProvider attribute, but it's specific to LDAP. Also, this line is a big fat hack. Actually, that could be said of most of the code associated with this PR. This is definitely a first iteration to see if the stated theory will increase performance. If it turns out that there are performance gains, I will refactor the code to be less kludgey.

Reply:
ok, that makes sense :)